website/docs/r/cdn_frontdoor_custom_domain.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+subcategory: "CDN"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cdn_frontdoor_profile_custom_domain"
+description: |-
+  Manages a Frontdoor Custom Domain.
+---
+
+# azurerm_cdn_frontdoor_profile_custom_domain
+
+Manages a Frontdoor Custom Domain.
+
+!>**IMPORTANT:** To delete a Custom Domain you must first disassociate the Custom Domain from the Frontdoor Route by removing the Custom Domains Resource ID from the Frontdoor Routes `cdn_frontdoor_custom_domain_ids` field. Once the Frontdoor Route has been updated, and the change to the Route has been provisioned to Azure, you may then remove the Custom Domain from your configuration file.
```

Comment:
in which case perhaps we need a "cdn_frontdoor_custom_domain_association` resource?

Reply:
Fixed.